src/config_web.py

```diff
@@ -90,84 +90,112 @@
 ]
 ANNOTATION_KWARGS = copy.deepcopy(ANNOTATION_KWARGS)
-ANNOTATION_KWARGS['*'].update(SPECIES_TYPEDEF)
-ANNOTATION_KWARGS['*']['_source']['strict'] = False
+ANNOTATION_KWARGS["*"].update(SPECIES_TYPEDEF)
+ANNOTATION_KWARGS["*"]["_source"]["strict"] = False
 
 QUERY_KWARGS = copy.deepcopy(QUERY_KWARGS)
-QUERY_KWARGS['*'].update(SPECIES_TYPEDEF)
-QUERY_KWARGS['*'].update(FIELD_FILTERS)
-QUERY_KWARGS['*']['_source']['default'] = DEFAULT_FIELDS
-QUERY_KWARGS['*']['_source']['strict'] = False
-QUERY_KWARGS['GET']['q']['translations'] = DATASOURCE_TRANSLATION_TYPEDEF
-QUERY_KWARGS['POST']['scopes']['translations'] = TRIMMED_DATASOURCE_TRANSLATION_TYPEDEF
-QUERY_KWARGS['GET']['include_tax_tree'] = {'type': bool, 'default': False}
-QUERY_KWARGS['POST']['scopes']['default'] = ["_id", "entrezgene", "ensembl.gene", "retired"]
-QUERY_KWARGS['POST']['q']['jsoninput'] = True
+QUERY_KWARGS["*"].update(SPECIES_TYPEDEF)
+QUERY_KWARGS["*"].update(FIELD_FILTERS)
+QUERY_KWARGS["*"]["_source"]["default"] = DEFAULT_FIELDS
+QUERY_KWARGS["*"]["_source"]["strict"] = False
+QUERY_KWARGS["GET"]["q"]["translations"] = DATASOURCE_TRANSLATION_TYPEDEF
+QUERY_KWARGS["POST"]["scopes"]["translations"] = TRIMMED_DATASOURCE_TRANSLATION_TYPEDEF
+QUERY_KWARGS["GET"]["include_tax_tree"] = {"type": bool, "default": False}
+QUERY_KWARGS["POST"]["scopes"]["default"] = [
+    "_id",
+    "entrezgene",
+    "ensembl.gene",
+    "retired",
+]
+QUERY_KWARGS["POST"]["q"]["jsoninput"] = True
 
 
 # *****************************************************************************
 # Elasticsearch Query Pipeline
 # *****************************************************************************
 ES_QUERY_BUILDER = "web.pipeline.MygeneQueryBuilder"
-AVAILABLE_FIELDS_EXCLUDED = ['all', 'accession_agg', 'refseq_agg']
+AVAILABLE_FIELDS_EXCLUDED = ["all", "accession_agg", "refseq_agg"]
 
 # *****************************************************************************
 # Endpoints Specifics & Others
 # *****************************************************************************
 
 # kwargs for status check
-STATUS_CHECK = {
-    'id': '1017',
-    'index': 'mygene_current'
-}
+STATUS_CHECK = {"id": "1017", "index": "mygene_current"}
 
-# This essentially bypasses the es.get fallback as in myvariant...
+# This essentially bypasses the es.get fallback as in myvariant ...
 # The first regex matched integers, in which case the query becomes against
 # entrezgeneall annotation queries are now multimatch against the following fields
-ANNOTATION_ID_REGEX_LIST = [(re.compile(r'^\d+$'), ['entrezgene', 'retired'])]
+base_regex_pattern = re.compile(r"^\d+$")
+base_field_scope = ["entrezgene", "retired"]
+
+# CURIE ID support
+BIOLINK_MODEL_PREFIX_BIOTHINGS_GENE_MAPPING = {
+    "NCBIGene": {"type": "gene", "field": "entrezgene"},
+    "ENSEMBL": {"type": "gene", "field": "ensembl.gene"},
+    "UniProtKB": {"type": "gene", "field": "uniprot.Swiss-Prot"},
+}
+gene_pattern_field_mapping = []
```

Comment:
maybe renaming this variable to `biolink_curie_regex_list`, just to be consistent with `ANNOTATION_ID_REGEX_LIST`?